WebKitPlus/ZenWebViewController.swift

```diff
@@ -15,10 +15,14 @@ open class ZenWebViewController: UIViewController {
         view.insertSubview(webView, at: 0)
 
         if #available(iOSApplicationExtension 11.0, *) {
-            let constraints = [webView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
-                               webView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
-                               webView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
-                               webView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor)]
+            webView.scrollView.contentInsetAdjustmentBehavior = .scrollableAxes
```

Comment:
https://developer.apple.com/documentation/uikit/uiscrollview/contentinsetadjustmentbehavior/automatic

> Content is always adjusted vertically when the scroll view is the content view of a view controller that is currently displayed by a navigation or tab bar controller. If the scroll view is horizontally scrollable, the horizontal content offset is also adjusted when there are nonzero safe area insets.

Since the webView (and its scrollView) is not a content view (root view) of the view controller, `.automatic` will not work. So let's specify `.scrollableAxes` explicitly.